langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/json/JsonSchemaElementHelper.java

```diff
@@ -185,7 +179,7 @@ public static Map<String, Object> toMap(JsonSchemaElement jsonSchemaElement) {
             if (jsonObjectSchema.description() != null) {
                 properties.put("description", jsonObjectSchema.description());
             }
-            properties.put("properties", toMap(jsonObjectSchema.properties()));
+            properties.put("properties", jsonObjectSchema.properties());
```

Comment:
Why is this change required? I can already see that behaviour for `JsonReferenceSchema` (recursion case) will be broken.